### PR TITLE
Added More App

### LINKED ISSUE #81: Create app for More
This app would be used to display items under nav-bar item `More`.

### Features:
Following views are to be added at /`more/`
* FAQs
* Hostel
* Students Corner
* Moodle
* Library
* Invited talks
* Placement

### LINKED ISSUE #107: Change Title of Careers Page
Change it from `Careers @ IIIT Vadodara` to `Careers`

### LINKED ISSUE #113: Fix Alignment of RTI.jpg
This alignment looks better than [original one](http://iiitvadodara.ac.in/rti.php).

err-align

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #81 
Fixes #107 
Fixes #113 
Deployed: http://13.59.1.49:8000/
username: test
password: test

### CODE REVIEW

File: .idea/Odyssy.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Monal5031) [inline .idea/Odyssy.iml]: Remove this file.

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<component name="InspectionProjectProfileManager">
```
Comment (Monal5031) [inline .idea/inspectionProfiles/Project_Default.xml]: Remove this file.

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Monal5031) [inline .idea/misc.xml]: Remove this file.

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Monal5031) [inline .idea/modules.xml]: Remove this file.

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Monal5031) [inline .idea/vcs.xml]: Remove this file.

File: .idea/workspace.xml [hunk 6]
```diff
@@ -0,0 +1,1035 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Monal5031) [inline .idea/workspace.xml]: Remove this file.

File: wsgi/odyssy/more/templates/more/faq.html [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"> <h3 class="panel-title"> <b>How to pay fees online?</b> </h3></div>
+            <div class="panel-body"> <br/>
+            <div class="embed-responsive embed-responsive-16by9"> <iframe allowfullscreen="" class="embed-responsive-item" src="https://www.youtube.com/embed/B7U3T_aaGCg"></iframe></div>
+            </div>
+        </div>
+        </div>
+    </div>
+>>>>>>> eb917ae2a3adc9475a977b3c0203f822e3300117
```
Comment (Monal5031) [inline wsgi/odyssy/more/templates/more/faq.html]: Remove this line.

File: wsgi/odyssy/more/templates/more/hostel.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
+{% extends 'basic/base.html' %}
+<<<<<<< HEAD
```
Comment (Monal5031) [inline wsgi/odyssy/more/templates/more/hostel.html]: Remove these unnecessary lines.

File: wsgi/odyssy/odyssy/urls.py [hunk 9]
```diff
@@ -33,7 +33,9 @@
     url(r'^people/', include('people.urls', namespace='people')),
     url(r'^tag/', include('tag.urls')),
     url(r'^careers/', include('careers.urls')),
+    url(r'^more/', include('more.urls', namespace='more')),
```
Comment (Monal5031) [inline wsgi/odyssy/odyssy/urls.py]: two times?

**Comment by yashLadha** [comment]
Add .idea to gitignore. Maybe we should update it more like omitting .swp files also. @Monal5031 Can u send a PR doing that.

**Comment by Monal5031** [comment]
@yashLadha Sure, I guess somehow the gitignore was changed, and .idea got shifted on previous line.

**Comment by geekyJock8** [comment]
**Alignment of Header is broken.** 

image
**Alignment of Gallery Name is broken.**

image

Also, width has become inconsistent all over the site. In some places, it's wider and in other places, it's narrower. I personally think that the new wider view looks better but make it consistent i.e. make it wider everywhere or if you want to revert back to the original.

**Comment by aashutoshrathi** [comment]
@geekyJock8 Actually, I was changing every page to wide view, but gallery title alignment was disturbed, so I left both gallery pages earlier.

**Comment by geekyJock8** [comment]
Inconsistency is removed (👍) as far as I can see but there are several UI bugs like the nav bar's width need to be adjusted to make it align with the width of the content section. Then there are bugs related to the gallery. If possible make a different PR for these width changes, it will make it easier to isolate bugs.

**Comment by Monal5031** [comment]
@yashLadha @Prakash2403 @geekyJock8 @mohitkyadav Please review this once.

**Comment by Monal5031** [comment]
gallery in `/more/hostel.html` are broken
screenshot 31

**Comment by aashutoshrathi** [comment]
Hostel Page Fixed, [Review again](http://13.59.1.49:8000/more/hostel.html)

**Pull request merged**